csharp.test/TestLogicalTypeRoundtrip.cs

```diff
@@ -143,6 +143,68 @@ public static void TestDateTimeRoundTrip(DateTimeKind kind, TimeUnit timeUnit)
             Assert.AreEqual(kind, kinds.First());
         }
 
+        [Test]
+        [NonParallelizable]
```

Comment:
I think it deserves a comment why this test shouldn't run together with other tests.